# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/gradient/reftest.list
# Commit: f733b37f9626
# Full Hash: f733b37f9626548ae83564a3fd43ddfa972977d9
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2021-04-22 21:31:57
# Regressor Bug: 1702228
# File Overlap Count: 1
# Description:
#   Bug 1702228 - Adjust reftests. r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112970
# ==============================================================================

diff -r 0919633e25e9 -r f733b37f9626 gfx/wr/wrench/reftests/gradient/reftest.list
--- a/gfx/wr/wrench/reftests/gradient/reftest.list	Thu Apr 22 10:34:55 2021 +0000
+++ b/gfx/wr/wrench/reftests/gradient/reftest.list	Thu Apr 22 10:34:55 2021 +0000
@@ -22,7 +22,7 @@
 fuzzy(1,20000) == linear.yaml linear-ref.yaml
 fuzzy(1,20000) == linear-reverse.yaml linear-ref.yaml
 
-fuzzy(1,15200) == linear-aligned-clip.yaml linear-aligned-clip-ref.yaml
+fuzzy(1,40000) == linear-aligned-clip.yaml linear-aligned-clip-ref.yaml
 
 platform(linux,mac) fuzzy(1,80000) == radial-circle.yaml radial-circle-ref.png
 platform(linux,mac) fuzzy(1,80000) == radial-ellipse.yaml radial-ellipse-ref.png